script/constants/environments.js

```diff
@@ -1,4 +1,5 @@
 module.exports = {
+  LOCALHOST: 'localhost',
   DEVELOPMENT: 'development',
   STAGING: 'staging',
   PRODUCTION: 'production',
```

Comment:
I default the `buildtype` to `localhost`, because it was otherwise the generated build directory was named `undefined`. This was from a previous revision to the build script. It didn't seem to break anything, but for clarity I added this. It played into how Heroku review instances were generated at `vagov-content`.